call.go

```diff
@@ -184,7 +183,7 @@ func Invoke(ctx context.Context, method string, args, reply interface{}, cc *Cli
 			continue
 		}
 		if EnableTracing {
-			c.traceInfo.tr.LazyLog(payload{reply}, true)
+			c.traceInfo.tr.LazyLog(&fmtStringer{"received: %v", []interface{}{payload{reply}}}, true)
```

Comment:
better yet, put a little bool in the payload struct and render the prefix lazily too.